back/engines/commercial/analysis/app/lib/analysis/q_and_a_method/base.rb

```diff
@@ -33,7 +33,8 @@ def initialize(question, *_args, **_kwargs)
 
     def execute(plan)
       task.set_in_progress!
-      question.update!(accuracy: plan.accuracy)
+      old_answer = question.answer
+      question.update!(accuracy: plan.accuracy, answer: '')
       question.insight.update!(inputs_ids: filtered_inputs.ids)
```

Comment:
Should we also store the old filtered inputs ids and restore that in case of an error?

Reply:
Yes, makes sense!